package/yast2-network.spec

```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-network
-Version:        3.4.0
```

Comment:
Strictly speaking, we don't need to increase version with every change to the storage-ng branch. We are not creating submit requests of this branch to any product, so there is no need to increase version . It can bring problems and confusion with no gain. Of course, the changelog entries are still needed, just leaving the number alone unless we need it in order to introduce a dependency in another package.